guardian-service/src/policy-engine/blocks/aggregate-block.ts

```diff
@@ -160,30 +179,28 @@ export class AggregateBlock {
         }
     }
 
+    /**
+     * Save documents
+     * @param ref
+     * @param doc
+     */
     async saveDocuments(ref: AnyBlockType, doc: any): Promise<void> {
         const vc = VcDocument.fromJsonTree(doc.document);
         const repository = getMongoRepository(AggregateVC);
-        const newVC = repository.create({
-            policyId: ref.policyId,
-            blockId: ref.uuid,
```

Comment:
this is broken the aggregate logic because blockId is not store in mongodb collection